composer.json

```diff
@@ -20,13 +20,13 @@
         "ext-mbstring": "*",
         "ext-openssl": "*",
         "doctrine/inflector": "^1.4|^2.0",
-        "dragonmantank/cron-expression": "^2.0",
+        "dragonmantank/cron-expression": "^2.3.1",
         "egulias/email-validator": "^2.1.10",
         "league/commonmark": "^1.3",
-        "league/flysystem": "^1.0.34",
+        "league/flysystem": "^1.1",
```

Comment:
This bumps the minimum supported version from `7.2.0` to `7.2.5`. https://laravel.com/docs/6.x states `>= 7.2.0`

Arguably BC break but I'm sure many would disagree.

Reply:
I think you only get this update by running `composer update` so the dependencies would get upgraded anyway?

Though maybe BC if you selectively upgrade dependencies.

Reply:
Also: bumping minimum versions in a stable release isn't a BC break: https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api

Reply:
The main point of the documentation is now wrong

Reply:
@bytestream what main point?